freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -107,6 +111,19 @@ public SyncOperationStatus getSyncOperationStatus(@NotNull String operationId) {
                 operationService.getOperationForAccountIdAndOperationId(accountId, operationId));
     }
 
+    @Override
+    public String getKeytab(@NotNull String environmentCrn, String targetUserCrn) {
```

Comment:
based on the expected future use-case, we should require the targetUserCrn field. i.e., mark it @NotNull.
As a design principle, we do not want to pull request arguments out of the actor or other metadata. All required information should be in the request itself. (yes, a bunch of the other APIs violate this but I'll fix this in the v2 API)

can we rename callerUserCrn to actorCrn?
we can keep the actorCrn.equals(targetUserCrn) check as a faux authorization check for now.

Reply:
Done. I've also renamed to actorCrn (I wonder if we should rename ThreadBasedUserCrnProvider -> ThreadBasedActorrCrnProvider).